### DISCUSSION TITLE
Nehirdeki Botlar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#168] [Zeka Sorusu]
Sabit hızla hareket eden iki güvenlik botu bir nehrin iki yakası arasında sefer yapmaktadır. Karşılıklı kıyılardan aynı anda hareket eden iki bot ilk kez yanyana geldiğinde birinci kıyıya 350 metre uzaklıktadır. Yollarına devam edip karşılarındaki kıyılara varır ve geri dönerler. (dönüşlerde vakit kaybı olmadığını varsayıyoruz) İkinci kez yanyana geldiklerinde ise diğer kıyıya 200 metre uzaklıktadırlar. Nehrin genişliğini bulunuz.

**Comment by Berkek007** [comment]
350*3= 1050 metre

1050-200= 850 metre -> nehrin genişliği

Tekneler sabit hızla gider ancak eşit hızda değiller
